### DISCUSSION TITLE
Is it possible to set custom icons?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [eza-community/eza#507] [Q&A, Howto, Help]
The issue for https://github.com/ogham/exa/issues/535 has been open for years and exa isn't developed anymore. Are there plans for custom icons if it isn't already supported?

**Comment by cafkafk** [comment]
Yes, there is! Right now it's blocked by #139, which is blocked by #195, which is blocked by #316, but slowly and surely we'll get there in due time!
